src/app/components/common/Header.tsx

```
@@ -27,13 +28,26 @@ const Header = () => {
           >
             Series
           </Link>
+          {/* <ThemeDropdown /> */}
+          <ThemeToggle />
+
           <button
             aria-label="검색"
             className="hidden sm:block text-sm text-neutral-600 hover:text-neutral-900 transition-colors cursor-pointer"
           >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)